CHANGELOG.md

```diff
@@ -8,14 +8,18 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 # Changelog
 
-The latest published Color release is [0.2.0](#020-2024-12-17) which was released on 2024-12-17.
-You can find its changes [documented below](#020-2024-12-17).
+The latest published Color release is [0.2.1](#021-2024-12-27) which was released on 2024-12-27.
+You can find its changes [documented below](#021-2024-12-27).
 
 ## [Unreleased]
 
 This release has an [MSRV][] of 1.82.
 
-###
+## [0.2.1][] (2024-12-27)
+
+This release has an [MSRV][] of 1.82.
+
+### Added
```

Comment:
There are missing brackets on the `#108` line below. Also 111 and 113 seem to be missing links. Otherwise looks good, but I haven't done a careful scan.

Reply:
These had already been fixed on `main` but this wasn't rebased on top yet.